src/components/popover-container/popover-container.component.tsx

```diff
@@ -164,6 +168,10 @@ export const PopoverContainer = ({
 
   const isOpen = isControlled ? open : isOpenInternal;
 
+  const reduceMotion = !useMediaQuery(
+    "screen and (prefers-reduced-motion: reduce)"
```

Comment:
```suggestion
    "screen and (prefers-reduced-motion: no-preference)"
```

**suggestion**: I may well be getting confused as I didn't get much sleep last night and I've only had one coffee so far 😅 , but if you look up how we're using this media query in `Loader` and `LoaderBar`, you'll see it's as I've suggested, which is nearly the opposite of what you've got here. ("nearly" because the `screen` check is in both - I'm not sure that's needed but I'm pretty sure it does no harm and in any case we've done that in the other components too and we should try to be consistent). And I'm pretty sure those other components are the ones that have this right - the logic seems to work that way for me, and also testing it out with the OS preference on and off, Loader/LoaderBar are working as intended. (I'd like to say which works better with PopoverContainer but unfortunately the animation there is so small I can't actually personally tell the difference!)

Reply:
Good point and thanks for noticing this. I'll get it changed so that it's consistent with `Loader` and `LoaderBar`.